tests/Test/Hspec/Expectations/JsonSpec.hs

```diff
@@ -9,10 +9,22 @@ import Prelude
 import Data.Aeson.QQ
 import Test.Hspec
 import Test.Hspec.Expectations.Json
+import Test.Hspec.QuickCheck (prop)
 
 spec :: Spec
 spec = do
+  describe "matchesJson" $ do
+    prop "always matches itself" $ \a -> a `matchesJson` a
+
   describe "shouldMatchJson" $ do
+    it "matches itself" $ do
```

Comment:
Do we need this given the property above?

Reply:
I wish, but `shouldMatchJson` doesn't use `matchesJson` under the hood

Reply:
Oh geeze. I wonder if there's a decent reason.